pkg/handlers/generic/lifecycle/ccm/nutanix/handler.go

```diff
@@ -174,3 +183,17 @@ func templateValuesFunc(
 		return b.String(), nil
 	}
 }
+
+func ipsToIgnore(nutanixConfig *v1alpha1.NutanixSpec) []string {
+	toIgnore := []string{nutanixConfig.ControlPlaneEndpoint.Host}
+	// Also ignore the virtual IP if it is set.
```

Comment:
Good catch @jimmidyson.
I added the new IP in addition to controlPlane IP but if we want we could also do just a single IP.
LMK what you think.

cc @supershal

Reply:
This looks good to me. Better to be safe than sorry IMO and add all IPs just in case.